src/pages/study-room/components/timer.tsx

```diff
@@ -1,15 +1,27 @@
 import { useEffect } from 'react';
+
 import { Button, Container } from '@mui/material';
 import KeyboardArrowRightIcon from '@mui/icons-material/KeyboardArrowRight';
+
 import { StepInfo } from '@/types/study-room.types';
+
+import useAudio from '../hooks/useAudio';
 import useStepInfo from '../hooks/useStepInfo';
 import useSockJSContext from '../hooks/useSockJSContext';
 
+import finishSound from '../../../assets/audio/finish.mp3';
+
 const Timer = (stepInfo: StepInfo) => {
   const { goToNextStep } = useSockJSContext();
 
   const { stepLabel, currentTime } = useStepInfo(stepInfo);
 
+  const { play, setSound } = useAudio();
```

Comment:
이런식으로 훅을 실행할 때 세팅해주는 인터페이스도 어떨까 싶네요~
```suggestion
  const { play, setSound } = useAudio({
    initialSound: finishSound
  });
```